Source/WebCore/inspector/NetworkResourcesData.cpp

```diff
@@ -256,10 +257,13 @@ void NetworkResourcesData::maybeDecodeDataToContent(const String& requestId)
     if (!resourceData->hasData())
         return;
 
-    m_contentSize += resourceData->decodeDataToContent();
-    size_t dataLength = resourceData->content().sizeInBytes();
-    if (dataLength > m_maximumSingleResourceContentSize)
-        m_contentSize -= resourceData->evictContent();
```

Comment:
is this no longer useful/desirable?  id think that if after decoding the resource is too big we'd wanna get rid of it

Reply:
ah, you are right, it is check also in ensureFreeSpace but it is not evicted in case the new size of resource is bigger than the limit. I will correct it.